can/notifier.py

```diff
@@ -13,57 +13,62 @@
 
 class Notifier(object):
 
-    def __init__(self, bus, listeners, timeout=None):
-        """Manages the distribution of **Messages** from a given bus to a
+    def __init__(self, bus, listeners, timeout=1):
+        """Manages the distribution of **Messages** from a given bus/buses to a
         list of listeners.
 
-        :param bus: The :ref:`bus` to listen too.
-        :param listeners: An iterable of :class:`~can.Listener`s
-        :param timeout: An optional maximum number of seconds to wait for any message.
+        :param bus: The :ref:`bus` or list of buses to listen to.
```

Comment:
I would say any iterable should do.

Reply:
It seems to be difficult to determine if a variable is iterable or not in an easy way. I'm not sure when one would need an iterable instead of a list.